src/components/Contact/Help.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
```

Comment:
Help.js: Code is clean and clearly presented. Any particular reason for the change from `import React, { Component } from 'react';` to `import React, { Component } from "react";`?